src/contracts/atlas/Atlas.sol

```diff
@@ -169,7 +169,7 @@ contract Atlas is Escrow {
 
         for (; winningSearcherIndex < solverOps.length;) {
             // Only execute solver meta tx if userOpHash matches
-            if (!auctionWon && userOpHash == solverOps[winningSearcherIndex].userOpHash) {
+            if (!auctionWon && solverOps[winningSearcherIndex].from != address(0)) {
```

Comment:
Do we need to check the 2nd condition here? Would solverOp.from ever be address(0)?

Reply:
Ah I see. Yes then looks good to me